meta-mender-tn-imx-bsp/recipes-bsp/u-boot/u-boot-tn-imx/edm-g-imx8mp/0001-configs-edm-g-imx8mp-add-Mender-support.patch

```diff
@@ -24,11 +24,10 @@ index 5d55972019..67f36b0da8 100644
 +CONFIG_SYS_REDUNDAND_ENVIRONMENT=y
 +CONFIG_BOOTCOUNT_LIMIT=y
 +CONFIG_BOOTCOUNT_ENV=y
-+CONFIG_BOOTCOUNT_BOOTLIMIT=3
  CONFIG_DM_GPIO=y
  CONFIG_TARGET_EDM_G_IMX8MP=y
  CONFIG_SPL_SERIAL_SUPPORT=y
-@@ -60,7 +65,6 @@ CONFIG_CMD_EXT4_WRITE=y
+@@ -60,7 +64,6 @@ CONFIG_CMD_EXT4_WRITE=y
```

Comment:
I think this is just a whitespace error.

Reply:
This part removes `CONFIG_ENV_IS_NOWHERE=y` (line 34 of the file).